includes/definitions.inc.php

```diff
@@ -1435,6 +1435,19 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+// Lenovo EMC (NAS)
+$os = 'lenovoemc';
```

Comment:
It's a somewhat customized version of Linux it seems - but if you ssh in to the box, it reports Linux (2.6.31.8). But SNMP doesn't match to Linux, which is why I gave it a special name. Thoughts?

Thanks!